scantools/scanners/navvis/navvis.py

```diff
@@ -677,6 +683,54 @@ def read_wifi(self):
                     wifi_measurements.append(wifi_measurement)
 
         return wifi_measurements
+    
+    def get_origin(self):
+        crs = UNKNOWN_CRS_NAME
+        qvec = [1, 0, 0, 0]
+        tvec = [0, 0, 0]        
+        if self.__origin_data:
+            orientation = self.__origin_data['orientation']
+            position = self.__origin_data['position']
+            qvec = [orientation['w'], orientation['x'], orientation['y'], orientation['z']]
+            tvec = [position['x'], position['y'], position['z']]
+        return crs, qvec, tvec
+    
+    def load_origin(self):
+        """Tries loading the NavVis origin from file.
```

Comment:
Maybe this comment should be moved to the definition of `parse_navvis_origin_file()`?